modules/ROOT/edits/install-self-managed.adoc

```diff
@@ -0,0 +1,259 @@
++
+*Windows:*
++
+----
+curl -L https://anypoint.mulesoft.com/runtimefabric/api/download/rtfctl-windows/latest -o rtfctl.exe
+----
++
+*MacOS (Darwin):*
++
+----
+curl -L https://anypoint.mulesoft.com/runtimefabric/api/download/rtfctl-darwin/latest -o rtfctl
+----
++
+*Linux:*
++
+----
+curl -L https://anypoint.mulesoft.com/runtimefabric/api/download/rtfctl/latest -o rtfctl
+----
+
+. Change file permissions for the `rtfctl` command-line utility:
++
+----
+sudo chmod +x rtfctl
+----
+
+== Step 4: Install Runtime Fabric
+
+[IMPORTANT]
+====
+The procedures in the section must be performed by an IT administrator.
+====
+
+After creating a Runtime Fabric in Runtime Manager and obtaining the activation data, install Runtime Fabric into your Kubernetes service using the `rtfctl` command-line utility.
+
+If your Kubernetes configuration is not located in the `\~/.kube/config` directory, set the `KUBECONFIG` environment variable before running `rtfctl`:
+----
+export KUBECONFIG=<path-to-kubeconfig>
+----
+
+
+. Validate that your Kubernetes environment is read for installation:
++
+----
+rtfctl validate <activation_data>
+----
++
+The `validate` option verifies that:
++
+* The Kubernetes environment is running.
+* All required components exist.
+* All required services are available.
++
+The `rtfctl` command-line utility outputs any incompatibilities with the Kubernetes environment.
+
+. Install Runtime Fabric:
++
+----
+rtfctl install <activation_data>
+----
++
+`<activation_data>` is the activation data obtained after creating the Runtime Fabric using Runtime Manager. During installation, the `rtfctl` utility displays any errors encountered.
+
+
+== Step 5: Insert the Mule License Key
+
+[IMPORTANT]
+====
+The procedures in the section must be performed by an IT administrator.
+====
+
+After the installation has completed succesfully, insert the Mule license key.
+
+. Base64 encode the new Mule `.lic` license file provided by MuleSoft:
++
+* On MacOS, run the following command:
++
+----
+base64 -b0 license.lic
+----
++
+* On Unix, run the following command:
++
+----
+base64 -w0 license.lic
+----
++
+* On Windows, a shell terminal emulator (such as cygwin) or access to a Unix-based computer is required.
++
+.. Transfer to your Unix environment if necessary.
+.. Run the following command to Base64 encode the license key:
++
+----
+base64 -w0 license.lic
+----
+
+. Insert the Mule license key:
++
+----
+rtfctl apply mule-license BASE64_ENCODED_LICENSE
+----
+
+. To verify the Mule license key has applied correctly, run:
++
+----
+rtfctl get mule-license
+----
+
+== Step 6: Configure the Ingress Resource Template
+
+[IMPORTANT]
+====
+The procedures in this section should be performed by an IT administrator.
+====
+
+If your ingress controller requires custom annotations and ingress class definition, follow the instructions in xref:custom-ingress-configuration.adoc[Defining a Custom Ingress Configuration].
+
+== Step 7: Validate Your Runtime Fabric
```

Comment:
This is not a step in the task, and it has no substeps of its own.
Suggest "Validation and Deployment Requirements"?